web/.env

```
@@ -1,3 +1,6 @@
 # Backend origin for API rewrites
 # Override per environment or set BACKEND_ORIGIN in deployment environment
-BACKEND_ORIGIN=http://localhost:5253
\ No newline at end of file
+BACKEND_ORIGIN=http://localhost:5253
+
+# Allow Node.js to connect to self-signed HTTPS backend
+NODE_TLS_REJECT_UNAUTHORIZED=0
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)